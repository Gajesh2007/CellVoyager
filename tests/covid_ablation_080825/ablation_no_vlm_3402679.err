
The following have been reloaded with a version change:
  1) python/3.12.1 => python/3.9.0

