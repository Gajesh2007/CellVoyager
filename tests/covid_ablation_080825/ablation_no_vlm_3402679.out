🚀 Running ablation test: no_vlm
   Self-critique: ✅
   VLM: ❌
   Analyses: 25
   Max iterations: 8

🧪 Testing no_vlm...
   Self-critique: ✅
   VLM: ❌
❌ Error during agent execution: 'deepresearch_summary'
💾 Results saved to: covid_ablation_080825/results_no_vlm.json
📊 NO_VLM Results:
   Final Success Rate: 0.00%
   Total Failure Rate: 0.00%
   Final Successful Cells: 0
   Total Failures: 0
   Total Attempts: 0
✅ Ablation test no_vlm completed!
